package.json

```diff
@@ -48,13 +49,20 @@
     "babel-plugin-module-resolver": "^5.0.2",
     "eslint": "^8.19.0",
     "eslint-plugin-prettier": "^5.5.4",
+    "husky": "^9.1.7",
     "jest": "^29.6.3",
+    "lint-staged": "^16.2.7",
     "patch-package": "^8.0.1",
     "prettier-eslint": "^16.4.2",
     "react-native-cli-bump-version": "^1.5.1",
     "typescript": "5.0.4"
   },
   "engines": {
     "node": ">=18"
+  },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx}": [
+      "npx eslint --fix"
```

Comment:
would we want to add a step that checks prettier as well?

Reply:
Prettier is included as an eslint rule, so that’s covered. Good call!